app/controllers/issues_controller.rb

```diff
@@ -0,0 +1,53 @@
+class IssuesController < ApplicationController
+  load_and_authorize_resource :project
+  load_and_authorize_resource through: :project
+
+  def show
+    @comment = Comment.new
```

Comment:
cancancan

Reply:
This is issues controller comment will not be created via cancancan

Reply:
discuss

Reply:
discussed